portal-impl/src/portal.properties

```diff
@@ -6785,6 +6785,11 @@
     #
     feature.flag.LPS-177194.type=release
 
+    #
+    # Env: LIFERAY_FEATURE_PERIOD_FLAG_PERIOD__UPPERCASEL__UPPERCASEP__UPPERCASES__MINUS__NUMBER1__NUMBER8__NUMBER0__NUMBER1__NUMBER5__NUMBER5__PERIOD_TITLE
+    #
+    feature.flag.LPS-180155.title=Display Publications with Data Set Display
```

Comment:
Hey @david-truong, is this title ok? Also, can you provide a description if you think it's necessary to have one for this feature flag?